configs/org.ppsspp.PPSSPP/config/ppsspp/PSP/SYSTEM/ppsspp.ini

```diff
@@ -454,3 +600,40 @@ ColorCorrectionSettingValue4 = 1.000000
 ScanlinesSettingValue1 = 1.000000
 ScanlinesSettingValue2 = 0.500000
 SharpenSettingValue1 = 1.500000
+[VR]
```

Comment:
there are a lot of extra config lines in this ini update that don't pertain to Achievements. Are they all needed?